internal/cmd/test/test.go

```diff
@@ -178,8 +179,9 @@ func (m mFlagRunner) Run() error {
 	return nil
 }
 
-func hasTestify(p string) bool {
-	cmd := exec.Command("go", "test", "-thisflagdoesntexist")
+func hasTestify(args []string) bool {
```

Comment:
the base of the checking command also should be a complete command (except the last wrong argument which causes an intentional error) so the execution can be guaranteed.